src/views/Admin/Charts/Edit.vue

```diff
@@ -321,11 +321,11 @@ export default {
     },
 
     nameState () {
-      return this.chart.name.length > 0 ? null : false
+      return this.chart && this.chart.name.length > 0 ? null : false
```

Comment:
Instead of adding it here and in handle state.
Could  you add the check in disableSave please. Thats the only place the nameState and handleState are used before chart loads.

And move the v-if="chart" to the b-container component (check out module/Edit for reference how its done there)